testsuite/src/main/java/io/netty/testsuite/transport/socket/SocketSslClientRenegotiateTest.java

```diff
@@ -130,7 +130,7 @@ public static Collection<Object[]> data() throws Exception {
     public void testSslRenegotiationRejected(final SslContext serverCtx, final SslContext clientCtx,
                                              final boolean delegate, TestInfo testInfo) throws Throwable {
         // BoringSSL does not support renegotiation intentionally.
-        assumeFalse("BoringSSL".equals(OpenSsl.versionString()));
+        assumeFalse("BoringSSL".equals(OpenSsl.versionString()) || OpenSsl.versionString().startsWith("AWS-LC"));
```

Comment:
The origin of this code is older than the `OpenSsl.isBoringSSL()` method. We can clean this up and use the new methods.

Reply:
Doing this here instead: https://github.com/netty/netty/pull/15451